Formula/scotch.rb

```diff
@@ -20,24 +21,32 @@ class Scotch < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "f93fe6e8bcdb31271f7d082915061d6fa2b9e415611a6f52929dbb45a48dd847"
   end
 
-  depends_on "open-mpi" => :build
+  depends_on "open-mpi"
 
   uses_from_macos "bison"
   uses_from_macos "flex"
   uses_from_macos "zlib"
 
   def install
-    cd "src"
-    (buildpath/"src").install_symlink "Make.inc/Makefile.inc.i686_mac_darwin10" => "Makefile.inc"
-    system "make"
-    system "make", "prefix=#{prefix}", "install"
+    chdir "src" do
+      (buildpath/"src").install_symlink "Make.inc/Makefile.inc.i686_mac_darwin10" => "Makefile.inc"
+
+      inreplace "Makefile.inc" do |s|
+        s.change_make_var! "CCS", ENV.cc
+        s.change_make_var! "CCP", "mpicc"
+        s.change_make_var! "CCD", "mpicc"
+      end
+
+      system "make", "scotch"
+      system "make", "ptscotch"
```

Comment:
```suggestion
      system "make", "scotch", "ptscotch"
```
Doesn't this work?

Reply:
> Doesn't this work?

It works. Also v6.1.2 is out. Do you want to bump the version right now, or let me do it in the PR?

Reply:
We can do it here.